receipt_upload/receipt_upload/dedup/run.py

```
@@ -0,0 +1,95 @@
+    s = rep["summary"]
+    print(f"[{args.env}] {rep['total_receipts']} receipts")
+    print(
+        f"  Tier 0 (exact, AUTO-MERGE):   {s['exact_groups']} groups, "
+        f"{s['exact_redundant_receipts']} redundant receipts"
+    )
+    print(
+        f"  Tier 1 (signature, REVIEW):   {s['signature_candidate_groups']} groups, "
+        f"{s['signature_candidate_receipts']} candidate receipts"
+    )
+
+    print(f"\n  sample Tier-0 exact groups:")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)